apps/strapi-cms/package.json

```diff
@@ -37,7 +37,7 @@
     "uuid": "1ae9c07f-4fbf-400a-9d32-bba36247aed0"
   },
   "engines": {
-    "node": ">=16.0.0",
+    "node": ">=18.0.0 <=20.x.x",
```

Comment:
It's not required, but it's the default value if you create a new Strapi project with version 4.24.2. I think setting the interval in this case is a good practice because it might help future developers when they update the node version